python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -2646,6 +2646,8 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: AtenRoundFloatModule())
 def AtenRoundFloatModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(5, 5, low = -3.0, high = 3.0))
+    # Test 'half to even' rounding behavior
+    module.forward(torch.FloatTensor([[0.5, 1.5], [-0.5, -1.5]]))
```

Comment:
nit: can you add this as a separate unit test (i.e. a separate nn.Module class) to match the style of the file?